AndelaOSP/art-backend
# #165377092 Allow posting and getting of asset invoice receipts

#### What does this PR do?
This PR enables an admin to view invoice_receipts of assets and gives a super_admin the ability to add an invoice_receipt to an asset

#### Description of Task to be completed?
Currently, there is no way of posting an asset along with its invoice_receipt. This PR, enables this.

#### How should this be manually tested?
1. On the browser, go to URL http://127.0.0.1:8000/api/v1/manage-assets/ and add an asset along with a pdf file in the invoice_receipt field. A 201 response should be returned. This field can also be updated as long as the user is super_admin.
2. A normal admin can view a link to an invoice_receipt when using the GET method to view assets.


#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/165377092
